libs/requests/src/requests.controller.ts

```diff
@@ -21,7 +21,7 @@ export class RequestsController {
   public async test(@Param('id') devideId: string): Promise<void> {
     try {
       await this.notifications.send({
-        deviceIds: [devideId],
+        regTokens: [devideId],
```

Comment:
Maybe call everything `registrationToken` ? I think we shouldn't have short names